src/Patch/Manager.php

```diff
@@ -7,171 +7,68 @@
-     */
-    public function applyAll()
-    {
-        $this->copyStaticFile();
-        $this->applyComposerPatches();
-        $this->applyHotFixes();
-    }
-
-    /**
-     * Copying static file endpoint.
-     * This resolves issue MAGECLOUD-314
-     *
-     * @return void
-     */
-    private function copyStaticFile()
-    {
-        $magentoRoot = $this->directoryList->getMagentoRoot();
-
-        if (!$this->file->isExists($magentoRoot . '/pub/static.php')) {
-            $this->logger->notice('File static.php was not found.');
-
-            return;
-        }
-
-        $this->file->copy($magentoRoot . '/pub/static.php', $magentoRoot . '/pub/front-static.php');
-        $this->logger->info('File static.php was copied.');
-    }
-
-    /**
-     * Applies patches from composer.json file.
-     * Patches are applying from top to bottom of config list.
-     *
-     * ```
-     *  "colinmollenhour/credis" : {
-     *      "Fix Redis issue": {
-     *          "1.6": "patches/redis-pipeline.patch"
-     *      }
-     *  }
-     *
-     * Each patch must have corresponding constraint of target package,
-     * in one of the following format:
-     * - 1.6
-     * - 1.6.*
-     * - ^1.6
-     *
-     * @return void
-     * @throws ShellException
-     * @throws FileSystemException
      */
-    private function applyComposerPatches()
+    public function apply(): void
     {
-        $patches = json_decode(
-            $this->file->fileGetContents($this->fileList->getPatches()) ?? '',
-            true
-        );
+        $this->logger->notice('Applying patches');
 
-        if (!$patches) {
-            $this->logger->notice('Patching skipped.');
-
-            return;
-        }
+        $command = 'php ./vendor/bin/ece-patches apply';
 
-        foreach ($patches as $packageName => $patchesInfo) {
-            foreach ($patchesInfo as $patchName => $packageInfo) {
-                if (is_string($packageInfo)) {
-                    $this->applier->apply($packageInfo, $patchName, $packageName, '*');
-                } elseif (is_array($packageInfo)) {
-                    foreach ($packageInfo as $constraint => $path) {
-                        $this->applier->apply($path, $patchName, $packageName, $constraint);
-                    }
-                }
-            }
+        if ($this->globalSection->get(GlobalSection::VAR_DEPLOYED_MAGENTO_VERSION_FROM_GIT)) {
+            $command .= ' --git-installation 1';
         }
-    }
-
-    /**
-     * Applies patches from root directory m2-hotfixes.
-     *
-     * @return void
-     */
-    private function applyHotFixes()
-    {
-        $hotFixesDir = $this->directoryList->getMagentoRoot() . '/' . static::HOTFIXES_DIR;
-
-        if (!$this->file->isDirectory($hotFixesDir)) {
-            $this->logger->notice('Hot-fixes directory was not found. Skipping.');
 
-            return;
+        try {
+            $this->shell->execute($command)->getOutput();
```

Comment:
Should this be logged? If not, why `getOutput()`?

Reply:
The output is logged always in Shell, removed `getOuptput`